src/Core/View.js

```diff
@@ -58,7 +59,38 @@ function View(crs, viewerDiv, mainLoop) {
 View.prototype = Object.create(THREE.EventDispatcher.prototype);
 View.prototype.constructor = View;
 
+function _defineLayerProperty(view, layer, propertyName, defaultValue) {
+    // todo : default value, test if setter and getter exist
+    var property = layer[propertyName] == undefined ? defaultValue : layer[propertyName];
+    Object.defineProperty(layer,
+        propertyName,
+        { get: () => property,
+            set: (newValue) => {
+                if (property !== newValue) {
+                    property = newValue;
+                    const event = { type: 'layer-changed' };
```

Comment:
And you could use the same semantic than you used in `GlobeControls`: old/new values.